### ISSUE TITLE
0.8.25 crash (Mac homebrew)

### ISSUE THREAD

**Issue by shawnyeager** [lra/mackup#1410]
`Traceback (most recent call last):
  File "/usr/local/bin/mackup", line 11, in <module>
    load_entry_point('mackup==0.8.25', 'console_scripts', 'mackup')()
  File "/usr/local/Cellar/mackup/0.8.25/libexec/lib/python3.7/site-packages/mackup/main.py", line 89, in main
    verbose)
  File "/usr/local/Cellar/mackup/0.8.25/libexec/lib/python3.7/site-packages/mackup/application.py", line 26, in __init__
    assert isinstance(files, set)
AssertionError`

**Comment by Xdeon** [comment]
Same here. Help appreciated.

**Comment by dmitry-ivanov** [comment]
Yeah, I have the same issue too.

**Comment by mnannola** [comment]
Same issue.  Using this with Dropbox if that matters.

`➜  ~ mackup backup
Mackup needs a directory to store your configuration files
Do you want to create it now? </Users/mnannola/Dropbox/Mackup> <Yes|No>Yes
Traceback (most recent call last):
  File "/usr/local/bin/mackup", line 11, in <module>
    load_entry_point('mackup==0.8.25', 'console_scripts', 'mackup')()
  File "/usr/local/Cellar/mackup/0.8.25/libexec/lib/python3.7/site-packages/mackup/main.py", line 89, in main
    verbose)
  File "/usr/local/Cellar/mackup/0.8.25/libexec/lib/python3.7/site-packages/mackup/application.py", line 26, in __init__
    assert isinstance(files, set)
AssertionError`

**Comment by zhongwangninja** [comment]
Same issue when executing `mackup backup` `mackup uninstall --f`
```
❯ mackup backup
Traceback (most recent call last):
  File "/usr/local/bin/mackup", line 11, in <module>
    load_entry_point('mackup==0.8.25', 'console_scripts', 'mackup')()
  File "/usr/local/Cellar/mackup/0.8.25/libexec/lib/python3.7/site-packages/mackup/main.py", line 89, in main
    verbose)
  File "/usr/local/Cellar/mackup/0.8.25/libexec/lib/python3.7/site-packages/mackup/application.py", line 26, in __init__
    assert isinstance(files, set)
AssertionError

❯ mackup uninstall --f
Traceback (most recent call last):
  File "/usr/local/bin/mackup", line 11, in <module>
    load_entry_point('mackup==0.8.25', 'console_scripts', 'mackup')()
  File "/usr/local/Cellar/mackup/0.8.25/libexec/lib/python3.7/site-packages/mackup/main.py", line 144, in main
    verbose)
  File "/usr/local/Cellar/mackup/0.8.25/libexec/lib/python3.7/site-packages/mackup/application.py", line 26, in __init__
    assert isinstance(files, set)
AssertionError
```

Extra environment info:
- mackup installed by brew
- mackup --versions: 0.8.25
- python set by pyenv 3.6.0

**Comment by misaka** [comment]
Looks like the issue is caused by the change in this commit: https://github.com/lra/mackup/pull/1402/commits/54e5d98ac84e9db10039cd2b28fe5c02b4460541#diff-cfe235ad3bb75c756c89b6a752e6dd4aR142. It seems that adding `sorted` there turns the set into an array.

```python
(Pdb) self.apps[name]['configuration_files']
{'.mackup', '.mackup.cfg'}
(Pdb) sorted(self.apps[name]['configuration_files'])
['.mackup', '.mackup.cfg']
```

which fails the assertion in the stack trace above:

```python
assert isinstance(files, set)
```

**Comment by misaka** [comment]
PR available #1411
